Resources/translations/fr/language.fr.xlf

```diff
@@ -0,0 +1,103 @@
+    <file source-language="en" target-language="en" datatype="plaintext" original="file.ext">
+        <body>
+            <trans-unit id="36aaf69db32866a9c2dd745ef9f7a643" resname="language.list">
+                <source>language.list</source>
+                <target>Langues</target>
+            </trans-unit>
+            <trans-unit id="d588dd1c265006f31dbb8368dda58226" resname="language.name">
+                <source>language.name</source>
+                <target>Nom de la langue</target>
+            </trans-unit>
+            <trans-unit id="6bb9217795bc1c5903f07476a0db6478" resname="language.code">
+                <source>language.code</source>
+                <target>Code de la langue</target>
+            </trans-unit>
+            <trans-unit id="1fdda1fb639ca79024d03e086790198c" resname="language.id">
+                <source>language.id</source>
+                <target>ID de la langue</target>
+            </trans-unit>
+            <trans-unit id="dc994226679dddc5cb5a96c45976850a" resname="language.enabled">
+                <source>language.enabled</source>
+                <target>Activée</target>
+            </trans-unit>
+            <trans-unit id="3a19ad2a3a91525fd4e43cfcb99176fe" resname="language.disabled">
+                <source>language.disabled</source>
+                <target>Désactivée</target>
+            </trans-unit>
+            <trans-unit id="bd9bc055699bd779f27ce1c3b2e0121d" resname="language.edit">
+                <source>language.edit</source>
+                <target>Éditer</target>
+            </trans-unit>
+            <trans-unit id="8a0af0dc53d7aae987d698237d39b11a" resname="language.new">
+                <source>language.new</source>
+                <target>Create a new language</target>
+            </trans-unit>
+            <trans-unit id="b2e01fb6617d17c162d59f0fb93da661" resname="language.remove.selected">
+                <source>language.remove.selected</source>
+                <target>Supprimer la selection</target>
+            </trans-unit>
+            <trans-unit id="a1388bfac224ede1622fd041e51384e0" resname="language.view.title">
+                <source>language.view.title</source>
+                <target>Langue: %languageName%</target>
+            </trans-unit>
+            <trans-unit id="967b7e408a5f43a03f44c7cf479855db" resname="language.create.title">
+                <source>language.create.title</source>
+                <target>Créer une nouvelle Langue</target>
+            </trans-unit>
+            <trans-unit id="f3c4115b0fe3b3ada04e69e775c2db2c" resname="language.edit.title">
+                <source>language.edit.title</source>
+                <target>Éditer la langue "%languageName%"</target>
+            </trans-unit>
+            <trans-unit id="007cd5845351d36a9ec70173d2b44bb2" resname="language.name.label">
+                <source>language.name.label</source>
+                <target>Language name:</target>
```

Comment:
*Nom de la langue :* (the space before *:* should be a non-breaking space, I'm unsure if you can put &#160; in translation files?)